pkg/api/volume_req.go

```diff
@@ -62,12 +62,18 @@ type VolCreateReq struct {
 	VolOptionReq
 }
 
+// VolOptionFlags is set of flags that allow/disallow setting certain kinds
+// of volume options.
+type VolOptionFlags struct {
+	AllowAdvanced     bool `json:"allow-advanced-options,omitempty"`
```

Comment:
@Madhu-1 @shtripat Please make a note of this API change for python mgmt bindings and tendrl/WA